pkg/dhcp6client/dhcp6client.go

```diff
@@ -92,7 +96,7 @@ func (p *buffer) remaining() []byte {
 
 func (c *Client) ReadReply() (*dhcp6.Packet, error) {
 	var err error
-	for i := 0; i < 5; i++ { // five attempts
+	for i := 0; i < c.attempts; i++ { // five attempts
 		pb := make([]byte, 1500)
```

Comment:
Based on RFC-791, Page 12: https://tools.ietf.org/html/rfc791#page-13,
do you think 576 bytes can be a more reasonable number of max size of a packet?